src/couch_prometheus/src/couch_prometheus_server.erl

```diff
@@ -177,24 +200,27 @@ get_io_stats() ->
     ].
 
 get_message_queue_stats() ->
-    QLenFun = fun(Name) -> message_queue_len(whereis(Name)) end,
-    Queues = lists:map(QLenFun, registered()),
+    QFun = fun(Name) -> {Name, message_queue_len(whereis(Name))} end,
+    Queues = lists:map(QFun, registered()),
+    QueueLens = lists:map(fun({_, Len}) -> Len end, Queues),
+    QueueLenByLabel = lists:map(fun({Name, Len}) -> {[{queue_name, Name}], Len} end, Queues),
     [
         to_prom(
-            erlang_message_queues, gauge, "total size of all message queues", lists:sum(Queues)
+            erlang_message_queues, gauge, "total size of all message queues", lists:sum(QueueLens)
         ),
         to_prom(
             erlang_message_queue_min,
             gauge,
             "minimum size across all message queues",
-            lists:min(Queues)
+            lists:min(QueueLens)
```

Comment:
Wondering if min queue length is useful at all. Amongst all the registered processes it's probably going to always be 0?

Reply:
I think it's a valid question, though `erlang_message_queue_min` already existed before this PR so it's technically a breaking change to remove it - perhaps something to schedule for removal in 4.x?